dask/dataframe/tests/test_arithmetics_reduction.py

```diff
@@ -661,9 +672,13 @@ def test_reductions(split_every):
             # runtime warnings; https://github.com/dask/dask/issues/2381
             assert_eq(dds.sem(split_every=split_every), pds.sem())
 
-        assert_eq(dds.std(ddof=0, split_every=split_every), pds.std(ddof=0))
-        assert_eq(dds.var(ddof=0, split_every=split_every), pds.var(ddof=0))
-        assert_eq(dds.sem(ddof=0, split_every=split_every), pds.sem(ddof=0))
+        with warnings.catch_warnings():
+            # dask.dataframe should probably filter this, to match pandas, but
```

Comment:
This is a bit hard. The warnings show up at runtime, so we would need to ensure that `np.errstate` is set correctly within each task calling the NumPy func.